Update interfaces/mediacapture-streams.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/3f10965/whatwg/idl/mediacapture-streams.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/463186803

foolip commented:
@lukebjerring leaving this one to you, looks like the status check claims regressions but then doesn't list any.

autofoolip committed:
Update interfaces/mediacapture-streams.idl

Source: https://github.com/tidoust/reffy-reports/blob/3f10965/whatwg/idl/mediacapture-streams.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/463186803

foolip commented:
These changes are from https://github.com/w3c/mediacapture-main/pull/550.

foolip commented:
There are quite a few tests depending on this file, but no regressions:
https://staging.wpt.fyi/results/?diff=&filter=ACU&product=chrome-72.0.3622.0+dev-linux-18.04%40774f67dcfc&product=chrome-72.0.3622.0+dev-linux-18.04%4030f6fb7652
https://staging.wpt.fyi/results/mediacapture-image/idlharness.window.html?product=firefox-65.0a1-linux-18.04%401abcb7058e&product=firefox-65.0a1-linux-18.04%4030f6fb7652&diff

@lukebjerring I love this thing more every day :)
